test/org/sagebionetworks/bridge/upload/UploadUtilTest.java

```diff
@@ -32,6 +34,151 @@
+                .withType(UploadFieldType.MULTI_CHOICE).withMultiChoiceAnswerList("foo").build();
+        UploadFieldSize fieldSize = UploadUtil.calculateFieldSize(ImmutableList.of(fieldDef));
+        assertEquals(5, fieldSize.getNumBytes());
+        assertEquals(1, fieldSize.getNumColumns());
+    }
+
+    @Test
+    public void calculateFieldSizeMultiChoiceWithManyChoice() {
+        UploadFieldDefinition fieldDef = new UploadFieldDefinition.Builder().withName("field")
+                .withType(UploadFieldType.MULTI_CHOICE)
+                .withMultiChoiceAnswerList("foo", "bar", "baz", "qwerty", "asdf").build();
+        UploadFieldSize fieldSize = UploadUtil.calculateFieldSize(ImmutableList.of(fieldDef));
+        assertEquals(25, fieldSize.getNumBytes());
+        assertEquals(5, fieldSize.getNumColumns());
+    }
+
+    @Test
+    public void calculateFieldSizeMultiChoiceWithOtherChoice() {
+        UploadFieldDefinition fieldDef = new UploadFieldDefinition.Builder().withName("field")
+                .withType(UploadFieldType.MULTI_CHOICE).withMultiChoiceAnswerList("foo", "bar")
+                .withAllowOtherChoices(true).build();
+        UploadFieldSize fieldSize = UploadUtil.calculateFieldSize(ImmutableList.of(fieldDef));
+        assertEquals(3010, fieldSize.getNumBytes());
+        assertEquals(3, fieldSize.getNumColumns());
+    }
+
+    @Test
+    public void calculateFieldSizeTimestamps() {
+        UploadFieldDefinition fieldDef = new UploadFieldDefinition.Builder().withName("field")
+                .withType(UploadFieldType.TIMESTAMP).build();
+        UploadFieldSize fieldSize = UploadUtil.calculateFieldSize(ImmutableList.of(fieldDef));
+        assertEquals(35, fieldSize.getNumBytes());
+        assertEquals(2, fieldSize.getNumColumns());
+    }
+
+    @Test
+    public void calculateFieldSizeMultipleFields() {
+        // Make fields
+        UploadFieldDefinition stringField = new UploadFieldDefinition.Builder().withName("string-field")
+                .withType(UploadFieldType.STRING).withMaxLength(50).build();
+        UploadFieldDefinition multiChoiceField = new UploadFieldDefinition.Builder().withName("multi-choice-field")
+                .withType(UploadFieldType.MULTI_CHOICE).withMultiChoiceAnswerList("foo", "bar")
+                .withAllowOtherChoices(true).build();
+        UploadFieldDefinition timestampField = new UploadFieldDefinition.Builder().withName("timestamp-field")
+                .withType(UploadFieldType.TIMESTAMP).build();
+        List<UploadFieldDefinition> fieldDefList = ImmutableList.of(stringField, multiChoiceField, timestampField);
+
+        // Execute and validate.
+        UploadFieldSize fieldSize = UploadUtil.calculateFieldSize(fieldDefList);
+        assertEquals(3195, fieldSize.getNumBytes());
+        assertEquals(6, fieldSize.getNumColumns());
+    }
+
+    // todo sum of multiple fields
```

Comment:
Still needs these tests or should comment be removed?

Reply:
Tests were added, but I forgot to remove the comment. Comment is now removed.